StrangeCalculatorTask/src/test/java/testsGoogle/MultiplyGoogleTests.java

```diff
@@ -0,0 +1,58 @@
+package testsGoogle;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+import strangecalculator.calculator.GoogleCalculator;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(Parameterized.class)
+public class MultiplyGoogleTests {
+
+    private Double x;
+    private Double y;
+    private Double expRez;
+    private GoogleCalculator calculator;
+
+    public MultiplyGoogleTests(Double x, Double y, Double expRez) {
+        this.x = x;
+        this.y = y;
+        this.expRez = expRez;
+    }
+
+    @Parameters
+    public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][]{
+                {1.0, 1.0, 1.0},
+                {-1.0, 1.0, -1.0},
+                {1.00, 0.00, 0.0},
+                {-99.00, -99.00, 9801.00},
+                {-4.000, 25.000, -100.000},
+                {-0.11, -0.99, 0.1089}
+        });
+    }
+
+    @Before
+    public void inicialisate() {
+        calculator = new GoogleCalculator();
+    }
+
+    @Test
+    public void testMultiplyMethod() {
+        Double actRez = calculator.multiply(x, y);
+        assertEquals(expRez, actRez);
+    }
+
+    @After
+    public void closeBrowser() {
```

Comment:
You implement the same method 4 times for each test class. It's a good idea to remove code duplication by moving this closing calculator to abstract class. The abstract class should be for each test with single 'after' method closeBrowser(), or for each calculator implementation.